### PR TITLE
F/interactable mainmerge

### LINKED ISSUE #35: Create interactables and obstacles
Create prefabs using the tiles prefab and add things that the player(s) can interact with

For example, see pallets from Dead by daylight

### PR DESCRIPTION
**Pull request by EvanDime**
added interactable to list of roads. I added the function to regenerate nodes but i am not sure how to test if it is working since the npc's seems to just run in a straight line and i can't see any gizmos. 

Please double check everything since I can't tell teh difference between main and mine. 

I have been a bit out of the loop and i think the only change is 4 npcs are hardcoded when the game runs?

### CODE REVIEW

File: Assets/Scripts/Level generation scripts/TriggerInteractableScript.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        }
+    }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        if (other.gameObject.name.ToLower().Contains("player") && Input.GetButtonDown("Jump"))
+        {
+            _triggered = true;
+        }
+    }
```
Comment (ClaudiaLapalme) [inline Assets/Scripts/Level generation scripts/TriggerInteractableScript.cs:33]: Is this even necessary if you have the same functionality in `OnTriggerStay` ?

**Review by ClaudiaLapalme — CHANGES_REQUESTED** [review]
One minor comment. Will test the PR in a little bit to double check that it works, but I have basically no comments concerning the code :D

File: Assets/Scripts/Level generation scripts/TriggerInteractableScript.cs (hunk 1 above)
Comment (EvanDime) [reply Assets/Scripts/Level generation scripts/TriggerInteractableScript.cs:33]: It's more of a safety net honestly. It makes the window to hit the button a bit larger. it's not necessary per say, but i put it for more consistency

Comment (ClaudiaLapalme) [reply Assets/Scripts/Level generation scripts/TriggerInteractableScript.cs:33]: Sounds good!

**Review by ClaudiaLapalme — APPROVED** [review]

**Review by tedgag — APPROVED** [review]
Looks great!

**Pull request merged**